aws-rds-cfn-common/src/main/java/software/amazon/rds/common/handler/Commons.java

```diff
@@ -93,4 +100,22 @@ public static <M, C> ProgressEvent<M, C> execOnce(
     public static Instant parseTimestamp(@NonNull String timestamp) {
         return ZonedDateTime.parse(timestamp).toInstant();
     }
+
+    public static <M, C> ProgressEvent<M, C> detectDrift(
+            final M inputModel,
+            final ProgressEvent<M, C> progress,
+            final ResourceTypeSchema schema,
+            final Logger logger
+    ) {
+        try {
+            final DriftDetector driftDetector = new DriftDetector(schema);
+            final Map<String, Mutation> drift = driftDetector.detectDrift(inputModel, progress.getResourceModel());
+            if (!drift.isEmpty()) {
+                logger.log(String.format("Resource drifted: %s", drift));
```

Comment:
May I ask that we utilize RequestLogger instead? 
Doing so would have the following benefits:
- Addition of customer and stack information as header
- Printing of Drift map object in JSON format, making it easier for CloudWatch to comprehend and parse. This streamlined process will facilitate mass querying of the logs.

Reply:
Done in both DBInstance and DBCluster. Thanks for the hint!